internal/expr/expr_test.go

```diff
@@ -44,43 +45,45 @@ type IntMap map[string]int
 type StringMap map[string]string
 
 var tests = []struct {
-	summary          string
-	query            string
-	expectedParsed   string
-	prepareArgs      []any
-	expectedPrepared string
+	summary        string
+	query          string
+	expectedParsed string
+	prepareArgs    []any
+	queryArgs      []any
```

Comment:
[Nit] Can we explicitly include the empty variable in the test cases, it makes them easier to read if they all have the same fields.

Reply:
I based this pattern off PL007 - (the Test Discipline section). In my opinion it makes it clearer, each test entry in the list of tests can be examined individually to see what is expected, if something is not expected or not inputted then it is left out to reduce visual clutter.

Reply:
Then leave it as it is, thanks for commenting about the reasoning.